HHS/Head-Start-TTADP
# [PROD] add goals filters and scopes to backend, fix misconfigured queue job

## Description of change

- Filters & scopes were added to the goals model. This is a backend-only change, advance work for putting a filter panel on the new goals page.
- The queue manager is now told to explicitly remove completed or failed jobs from the queue, again a backend-only change.

## How to test

- Upload files to a new activity report. Site doesn't crash. Approve and view activity report, confirm file has been uploaded.
- Confirm filters still work as expected

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-565
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-626

## Checklists

### Every PR


- [N/A] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [N/A] API Documentation updated
- [N/A] Boundary diagram updated
- [N/A] Logical Data Model updated
- [N/A] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
